css/css-anchor-position/anchor-position-dynamic-001.html

```diff
@@ -59,9 +59,9 @@
         data-offset-x=200 data-offset-y=100
         data-expected-width=300 data-expected-height=100></div>
   </div>
-<script>
+<script type="module">
 document.body.offsetTop;  // Force layout.
 container.classList.add('after');
-checkLayout('#target');
+await checkLayoutDelayed('#target');
```

Comment:
@jerivas Even though it's not strictly necessary, I added this `await` even when it's the last call -- just because it doesn't hurt, and it could avoid unintentional bugs if the test changes.